source/pages/Send/Confirm.tsx

```diff
@@ -132,7 +132,7 @@ export const SendConfirm = () => {
 
           const decryptPrivateKey = CryptoJS.AES.decrypt(
             String(activeAccount?.web3PrivateKey),
-            'encripted'
+            'encrypted'
```

Comment:
here we always need to pass a secret as the key for encryption/decryption, so you can use the wallet encrypted password from wallet controller